ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -218,8 +225,8 @@ Get Model Inference
     [Arguments]    ${model_name}    ${inference_input}    ${token_auth}=${FALSE}
     ...    ${project_title}=${NONE}
     ${self_managed} =    Is RHODS Self-Managed
-    ${url}=    Get Model Route via UI    ${model_name}
```

Comment:
same comment from the other PR, why? if you need to use CLI instead of UI, you could either create another keyword or adding a parameter and a new IF/ELSE branch in the original kw

Reply:
Reverted back the change